README.md

```diff
@@ -31,18 +29,3 @@ or
 ```bash
 npm install --save @lifi/types
 ```
-
-## How to make a release
-
-1. PR with changes
-2. Merge PR into main
-3. Checkout main
-4. `git pull`
-5. `yarn release`
-6. Make sure everything looks good (e.g. in CHANGELOG.md)
-7. `git push --follow-tags`
-8. Done
```

Comment:
Why is this removed?

Reply:
Should this be a part of internal documentation or the onboarding process? You don't usually see this in the README of the OSS projects 🙂

Reply:
Sure, you can add it to the  [internal Knowledge Base](https://www.notion.so/lifi/Knowledge-Base-75decf2b0e494555a35ad93a595a2086), Maybe add a new page called `Releasing Packages` or so?

Reply:
I added [Releasing Packages](https://www.notion.so/lifi/Releasing-Packages-b22e5e23cc2c482caa57ff977c5fa4fa) page 🙂